src/main/java/com/axis/system/jenkins/plugins/downstream/cache/BuildCache.java

```diff
@@ -38,13 +42,19 @@
  * @author Gustaf Lundh (C) Axis 2018
  */
 public class BuildCache {
-  private static final Logger LOGGER = LoggerFactory.getLogger(BuildCache.class.getName());
   private static final long GC_INTERVAL = TimeUnit.MINUTES.toMillis(10);
 
+  private static final org.slf4j.Logger Logger =
+      LoggerFactory.getLogger(BuildCache.class.getName());
```

Comment:
Nit, why explicit import?

Reply:
Naming conflict between the Class Logger and the Object Logger.

Reply:
Aha